fusestore/store/src/configs/config.rs

```diff
@@ -137,6 +137,14 @@ pub struct Config {
                       " Otherwise this argument is ignored.")
     )]
     pub id: NodeId,
+
+    #[structopt(
+        long,
+        env = "FUSE_STORE_LOCAL_FS_DIR",
+        help = "local dir for dfs storage and raft meta storage",
```

Comment:
Local fs dir is for storing files upload by users. 
All meta or raft data is in `meta_dir`

Reply:
ok，i will fix this